src/hooks/UrqlUseMutation.rei

```diff
@@ -4,3 +4,11 @@ let useMutation:
     UrqlTypes.hookResponse('response),
     unit => Js.Promise.t(UrqlClient.ClientTypes.operationResult),
   );
+
+let useDynamicMutation:
+  (~query: string, ~parse: Js.Json.t => 'response) =>
+  (
+    UrqlTypes.hookResponse('response),
+    {.. "variables": Js.Json.t} =>
```

Comment:
Is there any harm in defining this type as:

```suggestion
    UrqlTypes.request('response) =>
```

That would match the signature of calling `.make` exactly, even tho we never access those other two fields. Or is the idea to explicitly _not_ include those fields so they can't be accessed?

Reply:
I thought that since we only use the variables, we don't need to require that users provide the other fields. This way, if someone can't use graphql_ppx for some reason, they can do this:

```reason
executeMutation({"variables": {"var1": "hello", "var2": "world"}});
```
instead of
```reason
executeMutation({
  "variables": {"var1": "hello", "var2": "world"},
  "query": "",
  "parse": a => a
});
```

Reply:
Ah right you are, good shout!